fakeDataGen.js

```diff
@@ -1,20 +1,30 @@
-const faker = require('faker');
 const mongoose = require('mongoose');
 const fs = require('fs');
 
 // Output array
 const outputArray = [];
 
-// Seed so we can reuse this file and get the same results
-faker.seed(100);
-
 // Create 100 records
 const numRecords = 100;
 for (let i = 1; i <= numRecords; i++) {
-  const numOfImages = faker.random.number({ min: 1, max: 10 });
+  const imageIds = [
```

Comment:
nit: I think it would be better to have these ids in one line, so you can see it at a glance?

Reply:
Yup, I agree it looks weird with them all spaced like this. I went ahead and put them all on one line, and I also changed const name from `imageIds` to `picsumImageIds` to make it more clear what this is an array of. I also included a comment explaining :)